nixos/modules/services/networking/shadowsocks.nix

```
@@ -48,14 +64,27 @@ in
         '';
       };
 
+      package = mkPackageOption pkgs "Shadowsocks" {
+        default = "shadowsocks-libev";
+        example = "services.shadowsocks.package = pkgs.shadowsocks-libev";
```

I think `example` should be only the right-hand side.
```suggestion
        example = "pkgs.shadowsocks-libev";
```
That said, I don't think an example is really necessary here.